tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -273,7 +273,7 @@ function install_artifacts() {
 					esac
 					;;
 				*)
-					tdx_not_supported_warning
+					tdx_not_supported ${ID} ${VERSION_ID}
```

Comment:
This was introduced by commit a9720495deb2 . @fidencio can you confirm ?

Reply:
Yep, my bad :-/